### PR TITLE
fixed the indentation in _nodes.html.erb

### LINKED ISSUE #11019: Fix Indentation In _nodes.html.erb
### 💡Format this Issue Into a First Timers Issue
This has been marked as a good candidate for becoming a [first-timers-only issue](https://code.publiclab.org/#r=all) like these, meaning that it's **simple, self-contained**, and with some extra formatting✍️, could be a great entry point for a new contributor. If you're familiar enough with this code, please consider **reformatting it as a first-timers-only issue**, and then ping @publiclab/reviewers to get it labelled🏷. Or, if this is not your first time, **try to solve it yourself**🧑‍💻!

****

Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new** and looking through our general [help wanted](https://github.com/publiclab/plots2/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Claim the issue by commenting. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!
<details>
  <summary>💡 <i>Learn how to claim</i> 🙋</summary>
  <h3>Claiming an issue</h3>
  <p>Unless the issue is marked as reserved for someone, you can just say "I'd like to try this!" and then you've claimed it - no need to wait for someone to assign it to you. Just be sure you link your pull request (PR) to this issue so we can see where your solution is.</p>
  <p>And open one early if possible - even before you've completed it with additional commits - and others can help you figure out any issues you may face.</p>
</details>

- [ ] 📝 **Update** the file [_nodes.html.erb](https://github.com/publiclab/plots2/edit/main/app/views/spam2/_nodes.html.erb) in the `$plots2` repository (press the little pen Icon) and edit the line as shown below.

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
$(document).ready(function () {
-	var table = table_main("#node_table");
-	$("#all").click(function () { //select all button
+  var table = table_main("#node_table");
+  $("#all").click(function () { //select all button
    select_all();
  });

  var pageselect = localStorage.getItem('page-select') || '30';
  $("#pageselect").val(pageselect);
-	$("#batch-spam").bind('click', function (e) { //batch spam
-		batch_nav("batch_spam");
+  $("#batch-spam").bind('click', function (e) { //batch spam
+    batch_nav("batch_spam");

	});
-	$("#batch-publish").bind('click', function (e) { //batch publish
-		batch_nav("batch_publish");
+  $("#batch-publish").bind('click', function (e) { //batch publish
+    batch_nav("batch_publish");
	});
-	$("#batch-ban").bind('click', function (e) { //batch ban
-		batch_nav("batch_ban");
+  $("#batch-ban").bind('click', function (e) { //batch ban
+    batch_nav("batch_ban");
	});
-	$("#batch-unban").bind('click', function (e) { //batch unban
-		batch_nav("batch_unban");
+  $("#batch-unban").bind('click', function (e) { //batch unban
+    batch_nav("batch_unban");
	});
-	$("#delete-batch").bind('click', function (e) { // batch delete
-		batch_nav("batch_delete");
+  $("#delete-batch").bind('click', function (e) { // batch delete
+    batch_nav("batch_delete");
	});
-	$('#pageselect').change(function () {
-		pagination("#pageselect", "/spam2/filter/");
+  $('#pageselect').change(function () {
+    pagination("#pageselect", "/spam2/filter/");
	});
-	$('#spammed').on('click', function () { //spam filter
-		search_table("spammed", "/spam2/filter/");
+  $('#spammed').on('click', function () { //spam filter
+    search_table("spammed", "/spam2/filter/");
	});
-	$('#unmoderated').on('click', function () { // unmoderated filter
-		search_table("unmoderated", "/spam2/filter/");
+  $('#unmoderated').on('click', function () { // unmoderated filter
+    search_table("unmoderated", "/spam2/filter/");
  });
-	$('#published').on('click', function () { // published filter
-		search_table("published", "/spam2/filter/");
+  $('#published').on('click', function () { // published filter
+    search_table("published", "/spam2/filter/");
	});
-	$('#reset_all').on('click', function () { // reset all filter
-		search_table("all", "/spam2/filter/");
+  $('#reset_all').on('click', function () { // reset all filter
+    search_table("all", "/spam2/filter/");
  });
  $('#created').on('click', function () { // Created_at filter
-		search_table("created", "/spam2/filter/");
+   search_table("created", "/spam2/filter/");
	});
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://app.egghead.io/playlists/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there.

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance!

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it.

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below, and [reach out in our chatroom](https://publiclab.org/chat)!

### PR DESCRIPTION
**Pull request by mr-funtor**
-Adjusted the index to the file to make it more readable.

Fixes #11019

@VictoriaAjala @TildaDares @publiclab/reviewers

Beneath is a screenshot of the previous file before the present change.

Screenshot (62)

https://github.com/publiclab/plots2/suites/6071328877/artifacts/209946595

### CODE REVIEW

**Comment by mr-funtor** [comment]
Hi @TildaDares I have made the changes.

@publiclab/reviewers

**Review by TildaDares — CHANGES_REQUESTED** [review]
@mr-funtor Can you take a look at the file again? Some lines aren't indented properly.

**Comment by mr-funtor** [comment]
> @mr-funtor Can you take a look at the file again? Some lines aren't indented properly.

@TildaDares please what lines would you like me to adjust?

**Comment by TildaDares** [comment]
@mr-funtor From here you can see the lines that need adjustments. 

Screenshot 2022-04-20 at 08 41 58

**Comment by mr-funtor** [comment]
> @mr-funtor From here you can see the lines that need adjustments.
> 
> Screenshot 2022-04-20 at 08 41 58

@TildaDares thank you for your response. I would make another pull request to reflect the changes.

**Comment by mr-funtor** [comment]
@TildaDares  I have made the readjustment

**Comment by mr-funtor** [comment]
@TildaDares I have made the readjustments.

File: app/views/spam2/_nodes.html.erb [hunk 1]
```diff
@@ -1,45 +1,47 @@
 <script>
 $(document).ready(function () {
-	var table = table_main("#node_table");
-	$("#all").click(function () { //select all button
+  var table = table_main("#node_table");
+  $("#all").click(function () { //select all button
     select_all();
   });
+
```
Comment (TildaDares) [inline app/views/spam2/_nodes.html.erb:7]: @mr-funtor Can you remove this empty line? Everything else looks good. Thanks!!

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: app/views/spam2/_nodes.html.erb (hunk 1 above)
Comment (mr-funtor) [reply app/views/spam2/_nodes.html.erb]: @TildaDares I have removed the line. Thank you

**Review by TildaDares — APPROVED** [review]
Looks good @mr-funtor. Thank you!!

**Comment by TildaDares** [comment]
Congratulations @mr-funtor 🎉

**Comment by mr-funtor** [comment]
> Congratulations @mr-funtor 🎉

@TildaDares Thank you so much 🕺🕺🕺

**Pull request merged**